polliwog/polyline/test_polyline_object.py

```diff
@@ -106,6 +106,14 @@ def test_immutability():
     np.testing.assert_array_equal(polyline.v, vertices)
 
 
+def test_array_is_contiguous():
+    polyline = Polyline(
+        np.flipud(np.array([[5.5, 0.5, 0.0], [5.5, 0.75, 0.0], [5.5, 0.5, -0.5]])),
+        is_closed=True,
+    )
+    assert polyline.v.flags["C_CONTIGUOUS"] is True
```

Comment:
I think this needs a confidence check. i.e. that C_CONTIGUOUS is False on the v's before they are passed to the Polyline constructor.

Also it would be helpful to add some background info or a link to the original bug in tri-again. Did you manage to track that down?

Reply:
Fixed.  

I found the commit here, though it doesn't explicitly explain the original issue: https://github.com/lace/tri-again/commit/bd9f6bf30cbb6b10fb14ce28c71d6e5f4a9e55ac  Is that sufficient?

That does make me wonder if we should revert this line as well:
https://github.com/lace/tri-again/commit/bd9f6bf30cbb6b10fb14ce28c71d6e5f4a9e55ac#diff-bd3ca70f3147d5df94dfd3798324a4ce7a12c4ec0f4afc886d35893106e1eac0R79

Reply:
I think #272 is the thing to link to.

Ooh, yes, let's do that, now that we know Polyline will create something contiguous. (To ensure compatibility, the release that removes that call to `ascontiguousarray` should specify a version of polliwog >= the Polyline immutability change.)

Reply:
Great.  Added that link above.  The fix I was referring to was adding the False assertion here: https://github.com/lace/polliwog/pull/350/files#diff-d57a0f2ee3c0b00a2f8bd0e838c53a16ff0949ad9bd67d9842ed3bd76d978edbR111-R112

I added the the tri_again update to this https://github.com/lace/tri-again/pull/60/commits/2fc10a130761d848da9cdb19bfe6de696e2ed7ac since that's where the polliwog version is being bumped already.